start/lib/Model/Fleet.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Model;
+
+class Fleet
+{
+    private $id;
+    private $name;
+    private $team;
+    private $fleetShips;
+
+    public function __construct($name)
```

Comment:
I think it would be better to just have a `setName` function than just doing this in the constructor. You need the setter anyway.